builder/assets/rsyslog.conf

```diff
@@ -0,0 +1,107 @@
+# /etc/rsyslog.conf configuration file for rsyslog
+#
+# For more information install rsyslog-doc and see
+# /usr/share/doc/rsyslog-doc/html/configuration/index.html
+
+
+#################
+#### MODULES ####
+#################
+
+module(load="imuxsock") # provides support for local system logging
+module(load="imklog")   # provides kernel logging support
+#module(load="immark")  # provides --MARK-- message capability
+
+# provides UDP syslog reception
+#module(load="imudp")
+#input(type="imudp" port="514")
+
+# provides TCP syslog reception
+#module(load="imtcp")
+#input(type="imtcp" port="514")
+
+
+###########################
+#### GLOBAL DIRECTIVES ####
+###########################
+
+#
+# Use traditional timestamp format.
+# To enable high precision timestamps, comment out the following line.
+#
+$ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
+
+#
+# Set the default permissions for all log files.
+#
+$FileOwner root
+$FileGroup adm
+$FileCreateMode 0640
+$DirCreateMode 0755
+$Umask 0022
+
+#
+# Where to place spool and state files
+#
+$WorkDirectory /var/spool/rsyslog
+
+#
+# Include all config files in /etc/rsyslog.d/
+#
+$IncludeConfig /etc/rsyslog.d/*.conf
+
+#
+# Enable custom output channels to limit file sizes
+#
+$outchannel limauth,/var/log/auth.log,10485760
+$outchannel limsyslog,/var/log/syslog,10485760
+$outchannel limdaemon,/var/log/daemon.log,10485760
+$outchannel limkern,/var/log/kern.log,10485760
+$outchannel limlpr,/var/log/lpr.log,10485760
+$outchannel limmail,/var/log/mail.log,10485760
+$outchannel limmailinfo,/var/log/mail.info,10485760
+$outchannel limmailwarn,/var/log/mail.warn,10485760
+$outchannel limmailerr,/var/log/mail.err,10485760
```

Comment:
Do we need to handle the 'mail' message log? The main problems we have with types: syslog, kernel, messages.

Reply:
You're right, we don't. Still, I thought it'd be better to use output channels everywhere (basically a style concern).